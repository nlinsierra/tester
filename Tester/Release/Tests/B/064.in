3C 6C QD 6H 5D TC 3D 4C 7H QC 4H JS 5H 9D JH KH KD
