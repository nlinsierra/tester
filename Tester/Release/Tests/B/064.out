35
2C 5C 7C 8C 9C JC KC AC 2D 4D 6D 7D 8D TD JD AD 2H 3H 8H 9H TH QH AH 2S 3S 4S 5S 6S 7S 8S 9S TS QS KS AS
